backend/src/modules/doubles-tournament/core/domain/MatchId.ts

```diff
@@ -0,0 +1,12 @@
+export class MatchId {
+  private readonly TYPE = 'MatchId';
+
+  private constructor(readonly raw: string) {}
+
+  static from(tournamentId: string, matchNumber: number): MatchId {
+    if (tournamentId.length <= 0) {
+      throw new Error('TournamentId cannot be empty!');
+    }
+    return new MatchId(tournamentId + '-' + matchNumber);
```

Comment:
W sumie to lepiej zrobić to np. z podkreślinikiem `_` pomiędzy. Bo jeśli np. tournamentId jest UUIDem to ma `-` już wewnątrzi może być ciężko potem to w drugą stronę wyciągnąć. A się pewnie przyda przy tasku kończenia meczu.

Reply:
Jeśli ma to być ID to też elegancko byłoby z podkreśleniem